packages/date-picker/src/panel/date.vue

```diff
@@ -127,6 +127,13 @@
           @click="changeToNow">
           {{ t('el.datepicker.now') }}
         </el-button>
+        <el-button
+          size="mini"
+          type="warning"
+          class="el-picker-panel__link-btn"
+          @click="handleClear">
```

Comment:
`handleClear` can't be correct. What if user have already picked a date.

Reply:
Right. I forgot to take account into this scenario.